runtime/queries/passwd/highlights.scm

```diff
@@ -1,12 +1,12 @@
-(user) @namespace
+(user) @module
 
-(auth) @symbol
+(auth) @string.special.symbol
 
 (gecos) @string
 
-(home) @text.uri @constant
+(home) @string.special.url @constant
```

Comment:
```suggestion
(home) @string.special.path @constant
```
Should we remove the `@constant`?

Reply:
Yes. Double captures are never acceptable; either it's a constant or a custom `@string.special.path.builtin` (which I'd prefer to leave for user customization).